### DISCUSSION TITLE
Multi Dimensional Alert Setting at Datasource level

### DISCUSSION THREAD

**Discussion by SaiUmeshKumar** [grafana/grafana#67213] [Alerting]
Current Unified Alerting enables users to create multi dimensional alerting on all datasources if it's enabled at Grafana level. Can the setting be brought down to the datasource level so that in same grafana, we can have some datasources with multi dimensional setup and some without ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82742. Feel free to continue the discussion around this there. Thank you!
